content/docs/15.how-to-guides/synchonous-executions-api.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Synchronous Executions API
+icon: /docs/icons/tutorial.svg
+---
+
+There are use-cases wher you want to trigger the flow, and get the flow's output in the API's response. In other words, you want the Executions API to behave synchronously.
+
+## Executions API
+
+Firstly, let us take a quick look at the Executions API. Executions API is an API used to invoke a flow execution. Say you have the following flow:
+
+```yaml
+id: myflow
+namespace: company.team
+tasks:
+  - id: mytask
+    type: io.kestra.plugin.core.debug.Return
+    format: hello from kestra
+
+outputs:
+  - id: mydata
+    type: STRING
+    value: "{{ outputs.mytask.value }}"
+    description: return some data
+```
+
+You invoke this flow using the Executions API as follows:
+
+```
+% curl -X POST  http://localhost:8080/api/v1/executions/company.team/myflow
+```
+
+By default, the Executions API is asynchronous. It will invoke the execution of the flow, and return back with only the time at which the execution was invoked. The response you will get from the above API call will look like:
```

Comment:
```suggestion
By default, the Executions API is asynchronous. It will invoke the execution of the flow, and return immediately with a response that includes the Execution ID and the time at which the execution was created:
```